module/zfs/vdev_mirror.c

```diff
@@ -796,50 +865,90 @@ vdev_mirror_state_change(vdev_t *vd, int faulted, int degraded)
 	}
 }
 
+/*
+ * Return the maximum asize for a rebuild zio in the provided range.
+ */
+static uint64_t
+vdev_mirror_rebuild_asize(vdev_t *vd, uint64_t start, uint64_t asize,
```

Comment:
looks like `start` is no longer used.

Reply:
This was converted to a new vdev_op_t vector (`.vdev_op_rebuild_asize`) and `start` is used by the dRAID implementaiton `vdev_draid_rebuild_asize()`.  Are you suggesting adding an `/* ARGSUSED */` used comment here?

Reply:
ah, I didn't realize it is an ops vector.  I'd only add the ARGSUSED if it's required for a clean build.